earthpy/spatial.py

```diff
@@ -43,6 +43,9 @@ def extent_to_json(ext_obj):
 
 # Calculate normalized difference between two arrays
 # Both arrays must be of the same size
+# This could be confusing if one does not know that b2 is subtracted from b1. 
```

Comment:
@shwh2628 it is not appropriate to place additional commentary into a file. the goal is to REMOVE comments from a file like this not add more. 

If you wanted to adjust the structure of the function, you should submit a PR suggesting the band order be swapped in the code. Please REMOVE this change. 

this wold be a change you would have a discussion about with the package maintainers and if you looked closely at the issue you would see that discussion 

[here](https://github.com/earthlab/earthpy/issues/2) as an issue!